src/js/shared/constants.js

```diff
@@ -172,6 +172,8 @@ export const RSA_CONFIG = [
 
 export const RSA_RANGE_LENGTH = 4;
 
+export const exportFileName = "Settings.json";
```

Comment:
On JavaScript the naming convention for constants variables is uppercased, e.g. `EXPORT_FILE_NAME`